### PR TITLE
test testAngularDisplay failure

### PR DESCRIPTION
**Pull request by prabhjyotsingh**
### What is this PR for?
There are few times when Selenium test case "testAngularDisplay" fails. This to reproduce it on travis and fix it.

### What type of PR is it?
Bug Fix

### CODE REVIEW

**Comment by felixcheung** [comment]
Thanks - couple more instances in the last day:
https://s3.amazonaws.com/archive.travis-ci.org/jobs/109303040/log.txt
https://s3.amazonaws.com/archive.travis-ci.org/jobs/109303825/log.txt

This looks different though
https://s3.amazonaws.com/archive.travis-ci.org/jobs/109303816/log.txt

File: zeppelin-server/src/test/java/org/apache/zeppelin/AbstractZeppelinIT.java [hunk 1]
```diff
@@ -18,7 +18,9 @@
 package org.apache.zeppelin;
 
 
+import com.amazonaws.util.Base64;
```
Comment (felixcheung) [inline zeppelin-server/src/test/java/org/apache/zeppelin/AbstractZeppelinIT.java]: should we prefer apache.common instead? https://commons.apache.org/proper/commons-codec/apidocs/org/apache/commons/codec/binary/Base64.html

Comment (prabhjyotsingh) [inline zeppelin-server/src/test/java/org/apache/zeppelin/AbstractZeppelinIT.java]: Sure, I'll change this import, but this is something that I was about to ask as well;
https://github.com/apache/incubator-zeppelin/pull/719/files#diff-ee7042344a99f6984ef528dbbe372bc2R139
Is there any other way to take that file out from travis server ?

**Comment by prabhjyotsingh** [comment]
Found 2 issues:
 - AbstractFunctionalSuite.scala : we were never closing the webDriver
 - ZeppelinIT.java : this X-Path (//div[contains(@id,\"_text\") and @class=\"text\"]) seems to be performing better.

**Comment by corneadoug** [comment]
CI is green, looks good

**Comment by felixcheung** [comment]
@prabhjyotsingh I think this PR is just to trigger CI? Do you intend to include fixes for the 2 issues you have identified in this PR?

**Comment by felixcheung** [comment]
FYI - https://s3.amazonaws.com/archive.travis-ci.org/jobs/110282609/log.txt
```
02:08:26,726  INFO org.apache.zeppelin.WebDriverManager:172 - Download of firebug version: 31, successful
02:10:10,947 ERROR org.apache.zeppelin.integration.ZeppelinIT:199 - Exception in ZeppelinIT while testAngularDisplay 
org.openqa.selenium.ElementNotVisibleException: Element is not currently visible and so may not be interacted with
Command duration or timeout: 8 milliseconds
```

**Comment by prabhjyotsingh** [comment]
@felixcheung, Yes, I do intend these 2 get merged, as this is small improvement.
But I'm still trying to find why its failing only for "paragraph5Editor"

**Comment by prabhjyotsingh** [comment]
Have tried it multiple times, I think we can merge this. 
Only reason it failed in last few build was 
`No output has been received in the last 10 minutes, this potentially indicates a stalled build or something wrong with the build itself.`
At least it solves above two problem (https://github.com/apache/incubator-zeppelin/pull/719#issuecomment-184818151). 
Also, this will now take a ScreenShot and post it in logs, so next time if it fails for same we will have additional information to debug.

**Comment by felixcheung** [comment]
LGTM. merging if no more comment

**Comment by felixcheung** [comment]
if you see `No output has been received in the last 10 minutes` please let me know

**Comment by prabhjyotsingh** [comment]
@felixcheung sure if I see it after https://github.com/apache/incubator-zeppelin/pull/727 gets merged, I'll let you know.

**Comment by felixcheung** [comment]
In the post commit build, it looks like Selenium tests are failing because of "Connection refused" from screen shots. Any idea?
https://s3.amazonaws.com/archive.travis-ci.org/jobs/111391355/log.txt

**Comment by prabhjyotsingh** [comment]
Let me look into it, but by screen shot, it looks like spark is down.

    java.net.ConnectException: Connection refused
       at java.net.PlainSocketImpl.socketConnect(Native Method)
       at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:339)
       at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:200)
       at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:182)
       at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
       at java.net.Socket.connect(Socket.java:579)
       at org.apache.thrift.transport.TSocket.open(TSocket.java:182)
       at org.apache.zeppelin.interpreter.remote.ClientFactory.create(ClientFactory.java:51)
       at org.apache.zeppelin.interpreter.remote.ClientFactory.create(ClientFactory.java:37)
       at org.apache.commons.pool2.BasePooledObjectFactory.makeObject(BasePooledObjectFactory.java:60)
       at org.apache.commons.pool2.impl.GenericObjectPool.create(GenericObjectPool.java:861)
       at org.apache.commons.pool2.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:435)
       at org.apache.commons.pool2.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:363)
       at org.apache.zeppelin.interpreter.remote.RemoteInterpreterProcess.getClient(RemoteInterpreterProcess.java:155)
       at org.apache.zeppelin.interpreter.remote.RemoteInterpreter.getFormType(RemoteInterpreter.java:280)
       at org.apache.zeppelin.interpreter.LazyOpenInterpreter.getFormType(LazyOpenInterpreter.java:104)
       at org.apache.zeppelin.notebook.Paragraph.jobRun(Paragraph.java:215)
       at org.apache.zeppelin.scheduler.Job.run(Job.java:171)
       at org.apache.zeppelin.scheduler.RemoteScheduler$JobRunner.run(RemoteScheduler.java:326)
       at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
       at java.util.concurrent.FutureTask.run(FutureTask.java:262)
       at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(ScheduledThreadPoolExecutor.java:178)
       at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:292)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
       at java.lang.Thread.run(Thread.java:745)

download

**Pull request closed without merging**